02-s3/README.md

```diff
@@ -121,21 +121,30 @@ Add an object to your bucket:
 
 _How would you copy the contents of the directory to the top level of your bucket?_
 
+> Answer: you can use aws `s3 localdir S3://bucket --recursive`
+
 ##### Question: Directory Copying
 
 _How would you copy the contents and include the directory name in the s3 object
 paths?_
 
+> Answer: I couldn't find a way to do this with cp but sync worked with `aws s3 sync .  s3://stelligent-u-matthew.holmes.labs` 
+> Although that everything in this module directory. I couldn't find a way to copy just one whole directory intact
+
```

Comment:
`aws s3 cp --recursive local s3://my-bucket/local` or `aws s3 sync local s3://my-bucket/local`

You just need to append the directory name.